mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -104,4 +122,73 @@ protected void initializeMailboxManager() throws Exception {
         storeMailboxManager.setMessageSearchIndex(messageSearchIndex);
         storeMailboxManager.init();
     }
+
```

Comment:
As I understood ignore_above it does not affect the _source at all. It just prevent too long field to be neither indexed or stored.
But it will never touch the source.
So ok you fixe still work for some obscure reason, but this has nothing to do with the _source.